documentation/4.0/content/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -10,7 +10,7 @@ description = "Auxiliary images are an alternative approach for supplying a doma
 
 ### Introduction
 
-Auxiliary images are an alternative approach for including Model in Image model files,
+Auxiliary images are the top recommended approach for including Model in Image model files,
```

Comment:
Not sure if it's always the best, although it's certainly very often the best. I recall that there are customers that prefer dealing with one image that encapsulates installation and configuration, rather than multiple.

Reply:
@tbarnes-us If we are trying to get folks to use AIs (and we think it's the best and recommended approach), then it is least confusing to use that wording.

Reply:
done